src/kohlrahbi/__init__.py

```diff
@@ -27,22 +27,28 @@
 
 
 # pylint:disable=anomalous-backslash-in-string
-def get_valid_pruefis(list_of_pruefis: list[str], all_known_pruefis: Optional[list[str]] = None) -> list[str]:
+def get_valid_pruefis(
```

Comment:
da die Funktion nun nicht nur die pruefis zurückgibt, würde ich auch den Namen anpassen.
vll so
```suggestion
def get_pruefi_to_file_map(
```

Reply:
Mir ist unklar, wieso an dieser Stelle ein dictionary von Prüfis und deren Datei-Pfade übergeben wird.
Meine Entwickler-Erwartung an die Funktion wäre, dass man nur prüfus übergibt und nur die validen zurückbekommt.

Hätte eher eine zweite Funktion erwartet in der ich die validen prüfis eingebe und von dieser dann ein dictionary erwartet welche die Zuordnung prüfis zu Dateipfad beinhaltet.

Macht das Sinn?